data/brands/shop/bakery.json

```
@@ -425,6 +425,20 @@
         "shop": "bakery"
       }
     },
+    {
+      "displayName": "Bayne's",
+      "locationSet": {"include": ["gb"]},
```

```suggestion
      "locationSet": {"include": ["gb-sct"]},
```

You can use `gb-sct` location set for Scotland.

---

While Bayne's currently only operate in Scotland, there's no indication that they wouldn't expand to the rest of the UK, so not sure if it would be appropriate to limit the geographic scope within NSI, especially as there doesn't appear to be any other "Bayne's" in the database.

If by convention, the NSI would prefer the narrower scope until it's appropriate to expand it, I'll accept this suggestion.

---

> If by convention, the NSI would prefer the narrower scope until it's appropriate to expand it, I'll accept this suggestion.

I'd probably narrow the scope for now, we can always expand it later very easily and quickly, but if a different Bayne's appeared in England before this company expanded, we might find this entry being added by mistake before we can catch up.